### ISSUE TITLE
GeoJSON export, in the most recent version?

### ISSUE THREAD

**Issue by c-arthurs** [qupath/qupath-docs#21]
Hi, quick question (and thanks for making some really great software).
I saw that GeoJSON export has recently been added to the docs but on my current version, 0.2.3 on mac (which I think is the most recent), I have no `Object data…` option under File. 

So I also tried the script export but it seems that the function does not exist - 
``` groovy 
def annotations = getAnnotationObjects()
def path = "/Users/ca/Desktop/file.geojson"

// 'FEATURE_COLLECTION' is standard GeoJSON format for multiple objects
exportObjectsToGeoJson(annotations, path, "FEATURE_COLLECTION")
```

``` groovy 
ERROR: MissingMethodException at line 5: No signature of method: org.codehaus.groovy.jsr223.GroovyScriptEngineImpl.exportObjectsToGeoJson() is applicable for argument types: (ArrayList, String, String) values: [[Annotation (Polygon) (Tumor)], ./file.geojson, FEATURE_COLLECTION]
```
Am I just running an old version without realising it or is it going to be rolled out in the next release? Cheers

**Comment by petebankhead** [comment]
Hi https://qupath.readthedocs.io/en/stable/ is for v0.2.3, while https://qupath.readthedocs.io/en/latest is the for the not-yet-released v0.3.0. Not quite sure where you saw the `exportObjectsToGeoJson` docs, but it *should* only be visible under 'latest' - and by default any visitors to readthedocs should be directed to 'stable'.

In any case, you can export as GeoJSON from v0.2.3 with just a few more lines of code https://qupath.readthedocs.io/en/stable/docs/advanced/exporting_annotations.html#geojson

Or you can download a release candidate for v0.3.0 from https://github.com/qupath/qupath/releases

**Comment by c-arthurs** [comment]
Awesome, thought it would be that I was looking at the wrong version! Thanks for the quick reply
